npm/package.json

```diff
@@ -40,6 +40,8 @@
     "statements": 70
   },
   "dependencies": {
+    "@aws-sdk/client-dynamodb": "^3.259.0",
```

Comment:
It would be better to pin the packages to a specific version: ~~^3.259.0~~ -> 3.259.0